components/forms/CreateTxForm/index.tsx

```diff
@@ -84,7 +80,7 @@ const CreateTxForm = ({ router, senderAddress, accountOnChain }: CreateTxFormPro
         dataJSON: JSON.stringify(tx),
       });
 
-      router.push(`/multi/${senderAddress}/transaction/${transactionID}`);
+      router.push(`/${chain.registryName}/${senderAddress}/transaction/${transactionID}`);
```

Comment:
What happens in cases where multi chain selection is turned off via the env variable? E.g. we have https://multisig.tgrade.finance/ configured for only one chain. Do we have a value in that case too?

Reply:
So chain.registryName is empty string at the start of the app. Then it checks if it is on:
1. URL
2. localStorage
3. ENV file

If it's not found anywhere it would default to cosmoshub, but I guess tgrade would have the registry name properly set in the envfile?

Reply:
Ah, right, nice. NEXT_PUBLIC_REGISTRY_NAME is not yet set in the Tgrade deployment but we can add it easily. Thanks for the confirmation.